.github/workflows/build-and-push-client-container.yml

```diff
@@ -0,0 +1,49 @@
+name: Build and Push Image
+on: [ push ]
```

Comment:
So a new container will only be pushed when code is merged, but do we want/need a container PR revision? I don't so, not initially at least